[2022-12-03 14:49:46,907] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:12:00+00:00 [queued]>
[2022-12-03 14:49:48,272] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:12:00+00:00 [queued]>
[2022-12-03 14:49:48,273] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:49:48,504] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-03 14:49:48,504] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 14:49:50,958] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 18:12:00+00:00
[2022-12-03 14:49:51,240] {standard_task_runner.py:52} INFO - Started process 525 to run task
[2022-12-03 14:49:51,593] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T18:12:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmp1ycszgu7', '--error-file', '/tmp/tmpbv_2jc0q']
[2022-12-03 14:49:51,740] {standard_task_runner.py:80} INFO - Job 108: Subtask get_last_id
[2022-12-03 14:49:56,576] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T18:12:00+00:00 [running]> on host b41792e8e843
[2022-12-03 14:50:05,556] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T18:12:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T18:12:00+00:00
[2022-12-03 14:50:06,509] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-03 14:50:07,779] {python.py:173} INFO - Done. Returned value was: None
[2022-12-03 14:50:13,360] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T181200, start_date=20221203T144946, end_date=20221203T145013
[2022-12-03 14:50:14,418] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-03 14:50:17,587] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
